src/SDLRenderingWindow.cpp

```diff
@@ -461,3 +461,21 @@ void SDLRenderingWindow::OnConfigurationPropertyRemoved(const std::string& key)
         UpdateWindowTitle();
     }
 }
+
+SDL_GameController* SDLRenderingWindow::FindController() {
+    //Check for joysticks
+    if( SDL_NumJoysticks() < 1 )
+    {
+        poco_debug(_logger, "No joysticks connected");
+    }
+
+    //For simplicity, we’ll only be setting up and tracking a single
```

Comment:
If you want, you could also add a setting and an option in the settings dialog to let the user select a specific controller. Then select the stored one automatically on startup, e.g. via [its name](https://wiki.libsdl.org/SDL2/SDL_GameControllerNameForIndex) or [device path](https://wiki.libsdl.org/SDL2/SDL_GameControllerPathForIndex) (which should be map to the port it is connected to).

If you're not into Dear ImGui, I can also add this stuff at a later time.

Reply:
Didn't knew about Dear ImGui before I had a look at this project. Adding features to the GUI is a bit over my head at the moment (my available time is scrace). Actually I'm not even into C++ but this doesn't hinder me to tinker around :-)

Reply:
Sure, no problem! I've got the same time issue, but I'll try my best to spend a bit of it on projectM every week 😀 
I'll review the changes this weekend more thoroughly, but at a glance it all looks fine, so expect it to be merged soon!

Thanks again for the contribution!